src/components/CreateProposal/CreateProposalContainer.tsx

```diff
@@ -159,14 +180,22 @@ class CreateProposalContainer extends React.Component<IProps, IState> {
           </label>
           <input
             id='beneficiaryInput'
+            className={this.state.errors.beneficiary ? css.error : null}
             maxLength={42}
             onChange={this.handleChange}
+            onBlur={e => this.validate()}
```

Comment:
seems like you could build this into the onChange handler

Reply:
I think it's annoying to have it shout "invalid address!" at you while you type, don't you think?

Reply:
hmm, yes thats true, I forgot that onChange fires all the time